packages/main/src/themes/CalendarHeader.css

```diff
@@ -61,6 +61,17 @@
 	display:flex;
 }
 
+.ui5-calheader-middlebtn {
+	flex-direction: column;
+	align-items: center;
+	justify-content:center;
+}
+
+.ui5-calheader-btn-sectext {
+	color: #6a6d70;
```

Comment:
Please use the corresponding css variable instead of the color

Reply:
Done